### PR TITLE
Update search , manga detail , chapterlist

### PR DESCRIPTION
**Pull request by ponekub083**
Checklist:

- [x] Updated `extVersionCode` value in `build.gradle` for individual extensions
- [ ] Updated `overrideVersionCode` or `baseVersionCode` as needed for all multisrc extensions
- [ ] Referenced all related issues in the PR body (e.g. "Closes #xyz")
- [x] Added the `isNsfw = true` flag in `build.gradle` when appropriate
- [x] Have not changed source names
- [ ] Have explicitly kept the `id` if a source's name or language were changed
- [x] Have tested the modifications by compiling and running the extension through Android Studio

### CODE REVIEW

**Comment by ponekub083** [comment]
@alessandrojean  please review my code

**Comment by alessandrojean** [comment]
Don't need to ping me every PR, someone will always review your code eventually, myself included.

File: src/th/niceoppai/src/eu/kanade/tachiyomi/extension/th/niceoppai/Niceoppai.kt [hunk 1]
```diff
@@ -20,9 +22,9 @@ import java.util.Calendar
 import java.util.Locale
 import java.util.concurrent.TimeUnit
 
-class Niceoppai : ParsedHttpSource() {
+open class Niceoppai : ParsedHttpSource() {
```
Comment (alessandrojean) [inline src/th/niceoppai/src/eu/kanade/tachiyomi/extension/th/niceoppai/Niceoppai.kt:25]:
```suggestion
class Niceoppai : ParsedHttpSource() {
```

Don't need to be open as there's no inheritance from it.

**Review by alessandrojean — CHANGES_REQUESTED** [review]
Seems okay. Just change the suggestion.

**Comment by ponekub083** [comment]
@alessandrojean 

> Seems okay. Just change the suggestion.

it applied.

**Pull request merged**